README.md

```diff
@@ -72,7 +71,7 @@ PARAMETERS:
 
 Example:
 ```bash
-$ puppet task run os_patching::patch_server --params='{"reboot": true, "security_only": false}' --query="inventory[certname] { facts.os_patching.patch_window = 'Week3' and facts.os_patching.blocked = false and facts.os_patching.package_update_count > 0}"
+$ puppet task run os_patching::patch_server --params='{"reboot": "Patched", "security_only": false}' --query="inventory[certname] { facts.os_patching.patch_window = 'Week3' and facts.os_patching.blocked = false and facts.os_patching.package_update_count > 0}"
```

Comment:
Are Enums case sensitive? Regardless, I think the documentation would benefit from case consistency with `patched` instead of `Patched`.